Corona-Warn-App/src/test/java/de/rki/coronawarnapp/covidcertificate/person/core/PersonCertificatesStorageTest.kt

```diff
@@ -517,11 +517,11 @@ class PersonCertificatesStorageTest : BaseTest() {
                 	"settings": {
                         "{\"dateOfBirth\":\"20.10.2020\",\"familyNameStandardized\":\"llNN\",\"givenNameStandardized\":\"ffNN\"}": {
                           "lastSeenBoosterRuleIdentifier": "BRN-456",
-                          "lastBoosterNotifiedAt": 0,
+                          "lastBoosterNotifiedAt": 0.0,
                           "showDccReissuanceBadge": false,
-                          "lastDccReissuanceNotifiedAt": 0,
+                          "lastDccReissuanceNotifiedAt": 0.0,
                           "showAdmissionStateChangedBadge": false,
-                          "lastAdmissionStateNotifiedAt": 0
+                          "lastAdmissionStateNotifiedAt": 0.0
```

Comment:
Has the data type changed here?

Reply:
yes, it is not Java Instant not Joda Instant. Serialization is done by `JavaTimeModule` in jackson. not sure why is `0.0` not `0`.